cmd/juju/caas/remove.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package caas
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"github.com/juju/juju/cloud"
+
+	"github.com/juju/juju/api"
+	"github.com/juju/juju/api/base"
+	cloudapi "github.com/juju/juju/api/cloud"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/jujuclient"
+)
+
+var usageRemoveCAASSummary = `
+Removes a k8s endpoint from Juju.`[1:]
+
+var usageRemoveCAASDetails = `
+Removes the specified k8s cloud from the controller and user-defined
```

Comment:
Do we want to be 100% clear that we'll remove local reference from this client only? What if the user has been using several clients?

Reply:
This is a global issue though right? remove-cloud etc all have the same semantics. Maybe we should evolve something holistically to ensure consistent messaging in all cases?

Reply:
So, this is certainly affecting a few commands and I have gone back to all credential CRUD commands to highlight exactly what they are operating on - remote/controller stored information or local.

Since this boils own to individual calls, I am not sure how you'd approach this holiistically. If you have an idea, I am all ears :)

Meanwhile, please clarify to users. I can foresee confusion otherwise :)